chart/fluentbit-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{  .Values.metadata.name  }}
+  namespace: {{  .Values.metadata.namespace  }}
```

Comment:
The `namespace` should not be set, it will be set by helm if the helm running with `-n`.